app/components/ui/Weather.vue

```
@@ -45,12 +45,18 @@ onMounted(() => {
 
 <template>
   <div
+    <<<<<<< HEAD
```

ESLint

[error] Parsing error: unexpected-character-in-attribute-name.

(vue/no-parsing-error)